### ISSUE TITLE
Please publish this extension on open-vsx registry

### ISSUE THREAD

**Issue by apjyotirmay** [EmberExperts/ember-snippets#10]
Pretty much the title. Thank you <3

**Comment by Exelord** [comment]
Hey :) Could you explain the reasoning?

**Comment by apjyotirmay** [comment]
> Hey :) Could you explain the reasoning? 

Yes. Actually Open VSX store is an open source store for FOSS implementations of VSCode that has Microsoft's telemetry service removed (the one built from VSCode's FOSS code without MS's proprietary stuff)

If only it's not an inconvenience for you =)

**Comment by Exelord** [comment]
Interesting, can you use it as a VSCODE registry?

**Comment by apjyotirmay** [comment]
No, not really. There are licensing issues. VSCode uses Microsoft's registry, which doesn't allow any FOSS implementation of VSCode to use that marketplace.

And Microsoft doesn't allow using any other registry with their build of VSCode.

Feel free to close this issue if you feel it's not worth your time to publish to 2 different marketplaces. It's possible to use plugins from Microsoft's registry, but through some extra steps (you can't simply search it in VSCodium's [FOSS VSCode] extensions panel).

It's just that some of us like using a non-proprietary software which lacks telemetry. No pressure on you to cater to our beliefs :D

**Comment by Exelord** [comment]
Wow, never heard of that :) Would you like to use the package there or you are asking for others?

**Comment by apjyotirmay** [comment]
It would be nice to be able to find the extension directly from there. As of now, I download extensions not avail there from Microsoft store and then manually install them in Codium.

Asking it primarily for others because if someone decides to use Codium, they might not know they can manually install extensions from MS store, and they might not be able to find this one for Codium or other FOSS builds of VS Code at all
